tests/MongoDB.Driver.Tests/MongoCollectionImplTests.cs

```diff
@@ -258,6 +258,73 @@ public void Settings_should_be_set()
             findOperation.Sort.Should().BeNull();
         }
 
+        [Theory] // The only excluded option is simplified form with different database, because it is not supported by server
+        [InlineData(false, false, false)]
```

Comment:
nit: these arguments can be modified a bit to be more understandable if you add the whole `$merge` stage as a test argument

Reply:
Thanks for suggestion!
Done.